### PR TITLE
Add support for rotations, check in some rotations.

### PR DESCRIPTION
**Pull request by evankanderson**
Next step will be to be able to fetch the rotations live (in parallel) from URLs... and have a single URLs file that lists all the rotations in the community directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # www
-Party on the web!
 
+Party on the web!
```
Comment (evankanderson) [inline README.md:3]: I think this was `prettier`, I can put it back if you want.

File: kodata/rotations/1-serving.rotation [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Serving community contacts
+#
+# Begin metadata
+#@ title: Serving
+#@ slack: #serving-questions
+#@ slacklink: https://knative.slack.com/archives/C0186KU7STW
+
+2020-12-14T17:00:00Z | tcnghia
```
Comment (evankanderson) [inline kodata/rotations/1-serving.rotation:8]: I'm okay with other formats, but this was pretty easy to write a parser for and is is pretty easy to read and grok as a human.

File: site.go [hunk 3]
```diff
@@ -17,8 +23,9 @@ type envConfig struct {
 	Port     int    `envconfig:"PORT" default:"8080" required:"true"`
 }
 
+var env envConfig
```
Comment (evankanderson) [inline site.go:26]: I hoisted this into a global so it was easy to get to `DataPath` outside of main. Happy to encapsulate more if you want.

File: site.go [hunk 4]
```diff
@@ -78,6 +99,7 @@ type tier struct {
 }
 
 type onCall struct {
+	rotationFile   string `json:"-"`
```
Comment (evankanderson) [inline site.go:102]: Oops, didn't need this. Removing now.

File: site.go [hunk 5]
```diff
@@ -17,8 +23,9 @@ type envConfig struct {
 	Port     int    `envconfig:"PORT" default:"8080" required:"true"`
 }
 
+var env envConfig
+
 func main() {
-	var env envConfig
 	if err := envconfig.Process("", &env); err != nil {
```
Comment (n3wscott) [inline site.go:29]: we could even move this to an init func.

**Comment by n3wscott** [comment]
LGTM

Thanks @evankanderson !!!!

File: kodata/rotations/1-serving.rotation (hunk 2 above)
Comment (evankanderson) [reply kodata/rotations/1-serving.rotation:8]: I couldn't find a good package for this. Unfortunately, it seems like most of the systems for this like PagerDuty don't have configuration files, they have APIs that aren't really very human-config-file manageable.

I'm modelling this off escalator configs, for reference. 😉

**Pull request merged**